content/docs/deployment/csmoperator/drivers/powermax.md

```diff
@@ -405,7 +405,7 @@ Create a secret named powermax-certs in the namespace where the CSI PowerMax dri
         powermax-array-config.yaml: |
           # List of comma-separated port groups (ISCSI only). Example: PortGroup1, portGroup2 Required for iSCSI only
           X_CSI_POWERMAX_PORTGROUPS: ""
-          # Choose which transport protocol to use (ISCSI, FC, NVMETCP, auto or None) defaults to auto if nothing is specified
+          # Choose which transport protocol to use (ISCSI, FC, NVMETCP, auto) defaults to auto if nothing is specified
```

Comment:
You need to update the same V1,V2,v3 pages as well

Reply:
![image](https://github.com/user-attachments/assets/959c9500-44ee-4589-a244-2b56ad412f01)
Changed for v1, v2 and v3 .